docs/source/community/persons_of_interest.rst

```diff
@@ -149,14 +149,10 @@ C10 utils and operator dispatch
 -  Dmytro Dzhulgakov (`dzhulgakov <https://github.com/dzhulgakov>`__)
 -  (emeritus) Sebastian Messmer (`smessmer <https://github.com/smessmer>`__)
 
-ONNX <-> PyTorch
-~~~~~~~~~~~~~~~~
--  Negin Raoof (`neginraoof <https://github.com/neginraoof>`__)
--  Gary Miguel (`garymm <https://github.com/garymm>`__)
+PyTorch -> ONNX
+~~~~~~~~~~~~~~~
 -  Bowen Bao (`BowenBao <https://github.com/BowenBao>`__)
--  (emeritus) Lu Fang (`houseroad <https://github.com/houseroad>`__)
```

Comment:
keep them and move Negin to emeritus?

Reply:
No particular reason, just wanted to check since there is the tag `emeritus` and seems like the style for this doc. For me I'm ok either way. Sorry for not clicking the resolve earlier.

Reply:
(Sorry for repeated resolves/unresolved github UI is glitchy) Can we please move this (and discussion) to the separate PR. I though point of the emeritus title is to recognize previous contributions to the project, in in that sense removing people from it feels a bit wrong and I would appreciate more feedback on the subject. (Unless there are precedents of removing people from the list)